backend/src/main/java/ca/mcgill/purposeful/model/CollaborationConfirmation.java

```diff
@@ -0,0 +1,64 @@
+package ca.mcgill.purposeful.model;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.Id;
+import org.hibernate.annotations.GenericGenerator;
+
+/**
+ * The CollaborationConfirmation class, the model for regular user's collaboration confirmation to a
+ * request in the database
+ */
+@Entity
+public class CollaborationConfirmation {
+
+  // ------------------------
+  // CollaborationConfirmation Attributes
+  // ------------------------
+
+  @Id
+  @GeneratedValue(generator = "uuid")
+  @GenericGenerator(name = "uuid", strategy = "uuid2")
+  private String id;
+
+  @Column(nullable = true)
```

Comment:
I think we should leave the additionalContact information or the message non nullable, so that there is always some contents to display in the collaboration confirmation? Same for request below.

If we wanted to from the business logic perspective, we could have default values, eg. email and a generic message, so the user could have a choice to just send a confirmation. But I think its good to enforce the presence of content from the backend

Reply:
I put the additionalContact information as nullable since it would be something additional. The users already register with their emails, so it would already be accessible without them providing an additional information. I can make the message mandatory if it is better.

The idea of a default value can be interesting. For example, in the business logic/frontend, we can suggest additionalContact taken from previous requests or confirmation.

Reply:
Perfect that makes sense. The DTO that we send to the frontend for the contact information can check if there is additional info provided, and if not, use the email. Sounds good.